src/browser/vdiff.js

```diff
@@ -0,0 +1,10 @@
+import { executeServerCommand } from '@web/test-runner-commands';
+import { expect } from '@open-wc/testing';
+
+export async function screenshotAndCompare(elem, name, opts) {
+	const rect = elem.getBoundingClientRect();
+	const { pass, message } = await executeServerCommand('brightspace-visual-diff', { name, rect, opts });
+	if (!pass) {
+		expect.fail(message);
```

Comment:
This removes that `expected false to be true` comment, we now just get `AssertionError: No golden exists` or similar